zork.js

```diff
@@ -0,0 +1,329 @@
+//globals
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+
+class Player{
+    constructor(name,title){
+        this.name = name,
+        this.title = title
+    }
+    inventory = [];
+    drop = function(item){
+        delete this.inventory[item];
+    }
+    take = function(item){
+        this.inventory.push(item);
+    }
+    speak = function(words){
+        console.log(words);
+    }
+    showInventory = function(){
+        console.log(inventory);
+    }
+}
+
+class Location{
+    constructor(name,description,subLocations, items, isCurrentLocation, nextLocation){
+        this.name = name,
+        this.description = description,
+        this.subLocations = subLocations,
+        this.items = items,
+        this.isCurrentLocation = isCurrentLocation,
+        this.nextLocation = nextLocation
+    }
+    goToSubLocation(){ 
```

Comment:
Even though it's outside of the constructor it should still be `this.goToSubLocation(){...`